Gemfile

```diff
@@ -60,6 +60,7 @@ gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sshkey",                         "~>1.8.0",       :require => false
+gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
Makes sense, and I was unaware that is was wanted on the appliance and not just locally.  In that case, what you have here probably makes sense to solve that goal, and I don't have a better suggestion off hand.